bin/generate_history.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+from datetime import datetime, timedelta
+import random
+import json
+
+start_date = datetime(2020, 4, 15)
+end_date = datetime(2020, 6, 20)
+
+users = [
+    'U985L6R1M',
+    'U0ZC11HC7',
+    'U5HKU1Q0W',
+    'U0VCW825A'
+]
+
+ops_probs = (["++"] * 99) + ["--"]
+
+delta = end_date - start_date
+
+history = []
+
+for i in range(delta.days + 1):
+    day = start_date + timedelta(days=i)
+
+    n_events = random.randint(5, 10)
+
+    for t in range(n_events):
+        event_ts = day + timedelta(hours=9+t)
+        user = random.choice(users)
+        ops = random.choice(ops_probs)
```

Comment:
I think this has a 'weights' thing that allows you to specify a 0.99 and 0.01 chance of specific options, vs the '100 array size hack'

Reply:
Should have googled that before defaulting to my outdated smartass solution. https://docs.python.org/dev/library/random.html#random.choices

Anyway, this is quick hacky script I put together, don't have to change it.